content/collections/reference/metadata.md

```diff
@@ -17,174 +17,187 @@ As an example, see the [complete metadata file](https://github.com/OpenTermsArch
 
-It also aims to promote greater transparency and accountability of these powerful digital services.`
+It also aims to promote greater transparency and accountability of these powerful digital services.
+`
+    required=false
 >}}
-  
+
 {{< configOption
     name="dataset"
-    type="url"
-    description="URL to released versions dataset."
-    example="`https://github.com/OpenTermsArchive/pga-versions/releases`"
+    type="uri"
+    description="URL to the dataset releases."
+    example="https://github.com/OpenTermsArchive/pga-versions/releases"
+    required=false
 >}}
 
 {{< configOption
     name="declarations"
-    type="url"
-    description="URL to declarations repository."
-    example="`https://github.com/OpenTermsArchive/pga-declarations`"
+    type="uri"
+    description="URL to the declarations repository."
+    example="https://github.com/OpenTermsArchive/pga-declarations"
+    required=false
 >}}
 
 {{< configOption
     name="versions"
-    type="url"
-    description="URL to versions repository."
-    example="`https://github.com/OpenTermsArchive/pga-versions`"
+    type="uri"
+    description="URL to the versions repository."
+    example="https://github.com/OpenTermsArchive/pga-versions"
+    required=false
 >}}
 
 {{< configOption
     name="snapshots"
-    type="url"
-    description="URL to snapshots repository."
-    example="`https://github.com/OpenTermsArchive/pga-snapshots`"
+    type="uri"
+    description="URL to the snapshots repository."
+    example="https://github.com/OpenTermsArchive/pga-snapshots"
+    required=false
 >}}
 
 {{< configOption
-    name="logo"
-    type="url"
-    description="URL to the collection's logo. Optimized PNG transparent image (min width 240px)."
-    example="`https://opentermsarchive.org/images/collections/pga.png`"
+    name="donations"
+    type="uri"
+    description="URL to the donations page."
+    example="https://opencollective.com/opentermsarchive"
+    required=false
 >}}
 
 {{< configOption
-    name="donations"
-    type="url"
-    description="URL to the donations page."
-    example="`https://opencollective.com/opentermsarchive`"
+    name="logo"
+    type="uri"
+    description="URL to the collection's logo. Optimized PNG transparent image (min width 240px)."
+    required=false
 >}}
 
 {{< configOption
     name="trackingPeriods"
-    type="array of tracking periods objects"
-    description="Tracking periods object, see [Tracking periods](#tracking-periods) section."
+    type="array of objects"
+    description="trackingPeriods object, see [TrackingPeriods](#trackingperiods) section."
+    required=false
 >}}
 
 {{< configOption
     name="governance"
-    type="array of entity objects"
-    description="Entity object, see [Entity](#entity) section."
+    type="object"
+    description="Map of organizations involved in the collection's governance, with organization names as keys governance object, see [Governance](#governance) section."
```

Comment:
Some words seem to be missing here 🙂

Reply:
Which ones?

Reply:
Sorry for not having been more explicit. I don't understand this part:

> with organization names as keys governance object